server/stripToken.js

```diff
@@ -3,7 +3,7 @@ let JWT = require('./JWT')
 module.exports = async (req, res, next) => {
   try {
     let token = JWT.getToken(req)
-    await JWT.setTokenPayload(req, token)
+    if (token) await JWT.setTokenPayload(req, token)
```

Comment:
Make sure you handle the cases where token is passed along but is blank

Reply:
get token could return `''`

Reply:
is that affect on something? if I don't have a token or an empty string I shouldn't run `setTokenPayload`